doc/ipythonconsole.rst

```diff
@@ -2,91 +2,159 @@
 
-#. If connecting to a remote kernel over ``ssh``, check the appropriate box and type the full hostname you're connecting to (in the form :file:`{username}@{hostname}:{port-number}`).
-   Then, enter *either* :file:`{username}` 's password on the remote machine, or your user SSH keyfile (typically :file:`.perm`) (only one is needed to connect), and press :guilabel:`Ok`.
+Connect to a remote kernel
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   The port number is the one on which the SSH daemon (``sshd``) is running, typically 22 unless you or your administrator has configured it otherwise.
+To connect to a kernel on a remote machine,
 
-.. image:: images/console/console-dialog-connect.png
-   :alt: Connect to kernel dialog, requesting path and connection details
+#. Launch a Spyder kernel on the remote host if one is not already running, with ``python -m spyder_kernels.console``.
 
-For more technical details about connecting to remote IPython kernels, see the `Connecting to a remote kernel`_ page in the IPython Cookbook.
-Just remember to enter the appropriate details into Spyder's :guilabel:`Connect to an existing kernel` dialog instead of launching a new frontend on the client with ``--existing``.
+   .. image:: images/console/console-connect-remote-step1.gif
+      :alt: Staring a Spyder kernel on a remote machine
 
-.. _Connecting to a remote kernel: https://github.com/ipython/ipython/wiki/Cookbook:-Connecting-to-a-remote-kernel-via-ssh
+#. Copy the kernel's connection file (:file:`{jupyter/runtime/dir/path}/kernel-{pid}.json`) to the machine you're running Spyder on.
 
+   You can get :file:`{jupyter/runtime/dir/path}` by executing ``jupyter --runtime-dir`` in the same Python environment as the kernel.
+   Usually, the connection file you are looking for will be one of the newest in this directory, corresponding to the time you started the external kernel.
 
-.. _console-features:
+   .. image:: images/console/console-connect-remote-step2.gif
+      :alt: Using SCP to copy the connection file to the local machine
 
-==================
-Supported features
-==================
+#. Click :guilabel:`Connect to an existing kernel` from the :guilabel:`Consoles` menu, and browse for or enter the path to the connection file from the previous step.
 
-Any :guilabel:`IPython Console` in Spyder, internally or externally created, supports additional features including:
+   As a convenience, kernel ID numbers (e.g. ``1234``) entered in the connection file path field will be expanded to :file:`{jupyter/runtime/dir/path}/kernal-{id}.json` on your local machine, if you've copied the connection file there.
 
-.. image:: images/console/console-completion.png
-   :alt: Spyder IPython Console, with a popup list of code completion guesses
+   .. image:: images/console/console-connect-remote-step3.gif
```

Comment:
- I also suggest for this one use a smaller screen resolution and avoid zooming.
- Also, there's practically no pause when clicking the browse button. So I think it's fine to pause for a bit at that point to see that the explorer dialog is open due to that.

Reply:
Okay, I'll redo everything. It just will take some time.

As for the pause, I'm a little unsure where precisely you want it—you want me to freeze-frame the mouse hovered over it? With the button is depressed? Or what part, exactly? Thanks.

Reply:
Redone, with a pause